katalog/cert-manager/webhook/webhookvalidatingconfig.yml

```diff
@@ -1,89 +1,70 @@
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: cert-manager-webhook
+  labels:
+    app: webhook
+    app.kubernetes.io/name: webhook
+  annotations:
+    cert-manager.io/inject-ca-from-secret: "cert-manager/cert-manager-webhook-tls"
```

Comment:
```bash
$ kubectl get secret -n cert-manager cert-manager-webhook-tls cert-manager-webhook-ca
NAME                       TYPE                DATA   AGE
cert-manager-webhook-tls   kubernetes.io/tls   3      15m
cert-manager-webhook-ca    kubernetes.io/tls   3      23m
```
Both exists after a migration. I checked the content of both secrets and the ca.crt value is exactly the same.
I will try in a fresh installation (let's see if there are this two secrets in a fresh installation)

Reply:
Tried in a fresh (0.14.1) installation same behavior.

So my conclusion is....... it will work with both alternatives, but the one provided by the upstream project is `cert-manager-webhook-tls`.

Reply:
Perfect.